src/itest/java/com/iexec/sms/secret/compute/TeeTaskComputeSecretIntegrationTests.java

```diff
@@ -53,8 +53,7 @@
     private static final String APP_DEVELOPER_PRIVATE_KEY = "0x2fac4d263f1b20bfc33ea2bcb1cbe1521322dbde81d04b0c454ffff1218f0ed6";
     private static final String REQUESTER_PRIVATE_KEY = "0xb8e97e9e217a50dedbe3c0c4c37b85a85a10d4eb23fca6dbad55162cfbb1c450";
 
-    @Autowired
-    private ApiClient apiClient;
```

Comment:
Should we remove the `ApiClient`?

Reply:
The `ApiClient` class has been deleted, resolving this thread